recipes/LibriSpeech/ASR/transformer/hparams/train_hf_whisper.yaml

```diff
@@ -0,0 +1,146 @@
+# ################################
+# Model: Whisper (Encoder-Decoder) + NLL
+# Augmentation: TimeDomainSpecAugment
+# Authors: Adel Moumen 2022, Titouan Parcollet 2022
+# ################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1986
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/train_whisper/<seed>
+wer_file: !ref <output_folder>/wer.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# URL for the biggest Fairseq english whisper model.
+whisper_hub: openai/whisper-tiny
+language: english
+###
+# Uncomment this block if you prefer to use an english normalizer.
+# Please download the json english normalizer used by OpenAI here: 
+# https://github.com/openai/whisper/blob/main/whisper/normalizers/english.json
+#normalizer_path: !PLACEHOLDER
+
+# Data files
+#data_folder: !PLACEHOLDER # e.g., /path/to/LibriSpeech
+data_folder: !PLACEHOLDER # e,g./path/to/LibriSpeech
+# If RIRS_NOISES dir exists in /localscratch/xxx_corpus/RIRS_NOISES
+# then data_folder_rirs should be /localscratch/xxx_corpus
+# otherwise the dataset will automatically be downloaded
+# data_folder_rirs: !ref <data_folder>
+train_splits: ["train-clean-100", "train-clean-360", "train-other-500"]
+dev_splits: ["dev-clean"]
+test_splits: ["test-clean", "test-other"]
+skip_prep: False
+train_csv: !ref <data_folder>/train.csv
+valid_csv: !ref <data_folder>/dev-clean.csv
+test_csv:
+    - !ref <data_folder>/test-clean.csv
+    - !ref <data_folder>/test-other.csv
+
+ckpt_interval_minutes: 30 # save checkpoint every N min
+
+# Training parameters
+number_of_epochs: 1
+lr_whisper: 0.00003
+sorting: ascending
+auto_mix_prec: False
+sample_rate: 16000
+
+# With data_parallel batch_size is split into N jobs
+# With DDP batch_size is multiplied by N jobs
+batch_size: 12
+test_batch_size: 8
+
+# this value is only used for the searchers as the bos starting token. 
+timestamp_index: 50363 
+eos_index: 50257
+bos_index: 50258
+
+# Decoding parameters
+min_decode_ratio: 0.0
+max_decode_ratio: 0.1
```

Comment:
@TParcollet I needed to reduce the max_decode_ratio because of the positional encoding that throws an error when the tokens sequence is superior to a specific value (4XX). Does it seem correct?

Reply:
That is weird, you can let it like that but why is it happening?